tests/e2e/desktop-navigation-bar.test.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  openBrowser,
+  closeBrowser,
+  goto,
+  text,
+  click,
+  screenshot,
+  waitFor,
+  write,
+  press,
+} from "taiko";
+
+import { config } from "@src/config";
+
+describe("DesktopNavigationBar", () => {
+  jest.setTimeout(20000);
+
+  beforeAll(async () => {
+    await openBrowser({ args: ["--window-size=1440,1000"], headless: true });
+  });
+
+  afterAll(async () => {
+    await closeBrowser();
+  });
+
+  test("Log in with a desktop computer, use the navbar to go to the logins page and go back to home page", async (done) => {
+    try {
```

Comment:
Add `expect.assertions(6)` before the `try/catch`.

Reply:
OK fixed a32423e